### DISCUSSION TITLE
Getting automation's feedback to the user

### DISCUSSION THREAD

**Discussion by GregFdz** [Budibase/budibase#8931] [Help] [answered]
Hey
I'm reeeeeally stuck, right now. Ironically, I thought that this would be the easiest part of my app to build. I have a screen that triggers an automation. After the automation, I want to get the feedback ad change the display according to the result of the automation. And I can't find any option to do it.

Thanks for your help

**Comment by melohagan** [comment, marked as answer]
Hi @GregFdz 

Automations run async, thus there is no feedback. Perhaps in future we could add automation events that could push notifications to app users, but this is currently not available.

**Reply by GregFdz** [reply]
I wasn't thinking of a live interaction but if the user has a session with a session ID then maybe he he triggers an automation, there could be an instance id linked to his session ID and he could the recover the automation's output with them. Just like the automation logs do but more specific to an instance.
